python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2124,7 +2123,7 @@ def test_parquet_writer_statistics(tmpdir, pdf, add_nulls):
     gdf = cudf.from_pandas(pdf)
     if add_nulls:
         for col in gdf:
-            gdf[col] = gdf[col].set_mask(random_bitmask(len(gdf)))
+            gdf[col][np.random.choice([False, True], size=len(gdf))] = None
```

Comment:
My biggest regret so far in the design of draft PR #10034 was that I hadn't yet introduced a `cudf.testing` function to do this, and found myself re-implementing this ugly numpy logic many times over. I think it's hard to read (hard to parse by eye and requires some nontrivial understanding of `np.random.choice`). Thus, I'd like to give it a name, even though it's only one line of code. Something like this would be my suggestion:
```python
def set_random_null_mask_inplace(series, null_probability=0.5):
    probs = [null_probability, 1-null_probability]
    series[np.random.choice([False, True], size=len(series), p=probs)] = None
```

Reply:
I was going to wait to do this in a future PR, but since you requested it I went ahead and added a function. I didn't change its usage anywhere except here though; I'll leave that to you in #10034.

Reply:
I really like what you did in c01d0f2! Approving.